superset/assets/spec/javascripts/explorev2/components/SelectField_spec.js

```diff
@@ -21,13 +21,18 @@ describe('SelectField', () => {
     wrapper = shallow(<SelectField {...defaultProps} />);
   });
 
-  it('renders a FormControl', () => {
-    expect(wrapper.find(FormControl)).to.have.lengthOf(1);
+  it('renders a Select', () => {
+    expect(wrapper.find(Select)).to.have.lengthOf(1);
   });
 
   it('calls onChange when toggled', () => {
-    const select = wrapper.find(FormControl);
-    select.simulate('change', { target: { value: 50 } });
+    const select = wrapper.find(Select);
+    select.simulate('change', { value: 50 });
     expect(defaultProps.onChange.calledWith('row_limit', 50)).to.be.true;
   });
+
+  it('renders a Creatable for freeform', () => {
+    wrapper = shallow(<SelectField {...defaultProps} freeForm />);
+    expect(wrapper.find(Creatable)).to.have.lengthOf(1);
+  });
```

Comment:
it would be great to test that the correct value is displayed when one (or an array) is saved.